tools/image-builder/files_debootstrap/etc/systemd/system/haveged.service

```diff
@@ -0,0 +1,9 @@
+[Unit]
```

Comment:
Newer version of the haveged package provide their own unit file. You might consider borrowing the file from buster, or at least some of its settings.  https://salsa.debian.org/debian/haveged/blob/buster/debian/haveged.service

We should upgrade the VM image to buster one of these days...

Reply:
buster upgrade is coming... https://github.com/firecracker-microvm/firecracker-containerd/pull/324/commits/d56b94b91645a778792bd716d301731223db2643

Reply:
Will update

Reply:
Updated, only difference between the service files now is I inlined the args to the daemon instead of maintaining a separate env file (just seems simpler for us) and left out `apparmor.service systemd-random-seed.service systemd-tmpfiles-setup.service` from the `After=` parameters as I don't see any of those services under image-builder's rootfs